.buildkite/build_whl.sh

```diff
@@ -6,3 +6,4 @@ make dockerenvdist
 buildkite-agent artifact upload 'dist/*.whl'
 buildkite-agent artifact upload 'dist/*.zip'
 buildkite-agent artifact upload 'dist/*.tar.gz'
+buildkite-agent artifact upload 'dist/*.pex'
```

Comment:
This is a code smell, but I project that there'll be a refactoring in develop soon once `bake` is integrated, as that will require a refactoring of the Dockerfile.